source/NVDAObjects/__init__.py

```diff
@@ -821,7 +821,7 @@ def _get_statusBar(self):
 	def _get_isCurrent(self):
 		"""Gets the value that indicates whether this object is the current element in a set of related 
 		elements. This maps to aria-current. Normally returns False. If this object is current
-		it will return one of the following values: True, "page", "step", "location", "date", "time"
+		it will return one of the following values: "true", "page", "step", "location", "date", "time"
 		"""
 		return False

```

Comment:
Given this returns boolean `False` I think it would be best if we converted "true" to boolean `True`. Or perhaps better would be to return one off: None, 'true', 'page', etc